src/QldbHash.ts

```diff
@@ -19,6 +19,8 @@ const HASH_SIZE: number = 32
 
```

Comment:
Should we add @internal to HASH SIZE? I see it in the docs

![Uploading Screen Shot 2021-04-15 at 2.48.51 PM.png…]()

Reply:
It's not exported in the typescript definition though.